shell/server/pre-meteor.js

```diff
@@ -43,6 +43,12 @@ function isSandstormShell(hostname) {
   return (hostname === HOSTNAME || (DDP_HOSTNAME && hostname === DDP_HOSTNAME));
 }
 
+function hasPseudoTLD(hostname) {
+  // Returns true if this hostname ends in one of the major pseudo-TLDs.
+  let tld = hostname.split(".").pop();
+  return (tld === ("onion" || "i2p" || "gnu"));
```

Comment:
Perhaps it's a nitpick, but I was concerned that using `indexOf()` would return true for an hostname like `storm.onion.example.com`. The same should be true for `_.contains`.